db/migrate/20231122105957_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
- [ ] **[OPTIONAL]** There was a lot of issues when setting up your files. Kindly look into your migration. The order with which the migrations were created is wrong. You should create users, posts, likes and comments, before trying to add indexes on them.

  ![image](https://github.com/ahmidou09/blog_app/assets/36542986/13e75919-f8b1-411e-8b8a-e7a459c3a0af)